apps/expo/package.json

```diff
@@ -32,9 +32,9 @@
     "superjson": "1.9.1"
   },
   "devDependencies": {
-    "@acme/api": "*",
-    "@acme/eslint-config": "*",
-    "@acme/tailwind-config": "*",
+    "@acme/api": "^0.1.0",
+    "@acme/eslint-config": "^0.1.0",
+    "@acme/tailwind-config": "^0.1.0",
```

Comment:
what's wrong with "*" in your opinion? to me it quickly "clicks" that this is an internal package since it's not really versioned

Reply:
pnpm always changes it back to "^0.1.0" in my repos, I guess it also happened on the root package.json when you bumped deps.